elections.md

```diff
@@ -66,9 +66,23 @@ elections held. In the event of a question of company membership (for example
 evaluating independence of corporate subsidiaries) a majority of all
 non-involved Steering Committee members will decide.
 
+### Committee Conflict of Interest
```

Comment:
1. Are there known conflict(s) of interest between the Security Committee and Steering? (i.e. is it expected that a Security Committee member steps down if elected to Steering?)
2. Between this PR and https://github.com/kubernetes/community/pull/6243/ @tpepper - pick one place to put this information and link to it from the other. Let's a void duplicating content ❤️ 
3. Do we need to call out the _general_ idea of conflicts of interest, or are the cases so limited that we can list them out more clearly? (ie. if the only conflicts of interest are with CoCC and maybe Security, it'll be clearer to just list those out.)

Reply:
Agree on deduping content and referencing.

Steering/CoCC membership is the only current conflict I envision, since they both provide different types of oversight to the project. I don't think there's a general conflict between steering membership and any other committee membership (like the security committee).